controls/datagrid/theming-and-styles/group-header-styling.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Group Header Styling
+page_title: .NET MAUI DataGrid Documentation - Group Header Styling
+description: Learn how to style the Group Headers in a grouped DataGrid control.
+position: 2
+slug: datagrid-group-header-styling
+---
+
+# Group Header Styling for .NET MAUI DataGrid
+
+Telerik .NET MAUI DataGrid provides a `GroupHeaderStyle` property which defines the appearance of the group headers when the data is grouped.
+
+>To learn more about the grouping functionality of the DataGrid, go to [Grouping Overview]({%slug datagrid-grouping-overview%}) topic.
```

Comment:
The definite article is missing. Adding the article adds one more word, so I suggest rewriting it to "see the...", which keeps the expression two words long.
```suggestion
>To learn more about the grouping functionality of the DataGrid, see the [Grouping Overview]({%slug datagrid-grouping-overview%}) topic.
```